### DISCUSSION TITLE
👋🏼  Welcome Fellow Experts!

### DISCUSSION THREAD

**Discussion by metaskills** [metaskills/experts#11] [Announcements]
Thanks so much for y'alls interest in Experts.js. I've just enabled Discussions for us. Here are a few categories to get us started.

* [💡 Ideas](https://github.com/metaskills/experts/discussions/categories/ideas) - How can this framework evolve as a product. What key features are missing that make it work for you?
* [🙌🏼  Show & Tell](https://github.com/metaskills/experts/discussions/categories/show-and-tell) - How have you used Experts.js? If you can not share your story, drop a ❤️  on this discussion just to let folks know we have helped.
* [🙏 Q & A](https://github.com/metaskills/experts/discussions/categories/q-a) - We are all figuring out how to use Multiple AI Agent Systems effectively. Run into a hard spot with Experts.js? Ask!

**Comment by metaskills** [comment]
Neat, found out today we were mentioned in TLDR AI. Great newsletter if folks were new to various sources of great content.

https://tldr.tech/ai/2024-05-20

**Comment by metaskills** [comment]
These will be initially out of order as I find more. Here was a great video by WorldofAI. I especially liked how he covered the thread management.

https://www.youtube.com/watch?v=70_8odb-qYc

**Comment by metaskills** [comment]
Fourth tool in this YouTube video.

https://www.youtube.com/watch?v=yPm9kTZKguY

**Comment by metaskills** [comment]
Neat idea doing local with the Assistants API. https://github.com/transitive-bullshit/OpenOpenAI

**Comment by yashdhanore** [comment]
hi! does this support Azure OpenAI also?

**Reply by metaskills** [reply]
That is a great question. Maybe. My first assumption is that the OpenAI SDK support standard endpoint configs. But my second thought is even if they did this would/does Azure support the Assistants API? Can you take a look around and let me know. Happy to support simple config options if not already presently supported.

**Reply by yashdhanore** [reply]
i think Azure does support it [check this](https://learn.microsoft.com/en-us/azure/ai-services/openai/how-to/assistant#setting-up-your-first-assistant) but it is in python for now!
